docs/user-guide/downward-api/README.md

```diff
@@ -29,4 +29,25 @@ $ kubectl logs dapi-test-pod | grep POD_
 2015-04-30T20:22:18.568024817Z MY_POD_NAME=dapi-test-pod
 2015-04-30T20:22:18.568087688Z MY_POD_NAMESPACE=default
 2015-04-30T20:22:18.568092435Z MY_POD_IP=10.0.1.6
-```
\ No newline at end of file
+```
+
+## Example of environment variables with container resources
+
+Use the [`dapi-container-resources.yaml`](dapi-container-resources.yaml) file to create a Pod
+with a container that consumes the downward API exposing the container's resources.
```

Comment:
Is it really correct to say that a container is consuming the downward API? That sounds off to me.